kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -7,7 +7,7 @@ The operator provides sample scripts to start up or shut down a specific Managed
 These scripts can be helpful when scripting the life cycle of a WebLogic Server domain. For information on how to start, stop, restart, and scale WebLogic Server instances in your domain, see [Domain Life Cycle](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle).
 
 #### Scripts to start and stop a Managed Server
-The `startServer.sh` script starts a Managed Server either by increasing the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or by updating the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
+The `startServer.sh` script starts a WebLogic Server in a domain. For clustered Managed servers, it either increases the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or updates the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. For the administration server, it updates the value of `spec.adminServer.serverStartPolicy` attribute of the domain resource. For non-clustered Managed servers, it updates the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
```

Comment:
Managed servers -> Managed Servers
it either increases -> either it increases
administration server -> Administration Server
value of -> value of the
Managed servers -> Managed Servers